build.bat

```diff
@@ -984,6 +988,9 @@ GOTO:EOF
 	echo.    --cmake  path                Path to the CMAKE executable. If this
 	echo.                                 parameter is not present, Cmake variable
 	echo.                                 should be available from your $PATH variable.
+	echo.    --add-cmake-args <s>         Additional defines and arguments that need to
```

Comment:
Same as build.sh: I think this would be more clear if it said "Additional defines and arguments that will be passed to the cmake executable..." Since on many platforms you will not need these arguments. I could see "need to" causing confusion on whether this is a required argument.

Reply:
Done